Cloud Build failing for header-checker-lint

```
Step #0 - "build": src/header-checker-lint.ts:25:28 - error TS7016: Could not find a declaration file for module 'minimatch'. '/workspace/packages/header-checker-lint/node_modules/minimatch/minimatch.js' implicitly has an 'any' type.
Step #0 - "build": Try `npm install @types/minimatch` if it exists or add a new declaration (.d.ts) file containing `declare module 'minimatch';`
Step #0 - "build": 
Step #0 - "build": 25 import * as minimatch from 'minimatch';
Step #0 - "build": ~~~~~~~~~~~
```

Not sure why it's passing CI, but failing here.

--- a/packages/header-checker-lint/package.json
+++ b/packages/header-checker-lint/package.json
@@ -28,6 +28,7 @@
   "dependencies": {
     "@octokit/rest": "^16.28.3",
     "gcf-utils": "^0.6.0",
+    "minimatch": "^3.0.4",
     "probot": "^9.3.0"
   },
   "devDependencies": {
@@ -36,6 +37,7 @@
     "@types/express": "^4.17.0",
     "@types/ioredis": "^4.0.13",
     "@types/lru-cache": "^5.1.0",
+    "@types/minimatch": "^3.0.3",
     "@types/mocha": "^5.2.7",
     "@types/nock": "^10.0.3",
     "@types/node": "^12.6.1",

